host-bmc/dbus/custom_dbus.cpp

```diff
@@ -105,9 +106,17 @@ void CustomDBus::updateItemPresentStatus(const std::string& path,
         // Hardcode the present dbus property to true
         presentStatus.at(path)->present(true);
 
-        // Set the pretty name dbus property to the filename
-        // form the dbus path object
-        presentStatus.at(path)->prettyName(ObjectPath.filename());
+        if (prettyName.empty())
```

Comment:
if it is an std::optional string, then you can just check if the value is present  - rather than checking for an string which is empty(), then use it ..otherwise use the ObjectPath.filename().

Reply:
Done